main/main.cpp

```diff
@@ -429,6 +431,49 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 			print_line(get_full_version_string());
 			goto error;
 
+		} else if (I->get() == "-v" || I->get() == "--verbose") { // verbose output
+
+			OS::get_singleton()->_verbose_stdout = true;
+		} else if (I->get() == "--quiet") { // quieter output
+
+			quiet_stdout = true;
+#ifndef SERVER_ENABLED
+		} else if (I->get() == "--audio-driver") { // audio driver
```

Comment:
I would leave the audio and video driver options available. They do work and provide useful information:
```
  --audio-driver <driver>          Audio driver ('Dummy').
  --video-driver <driver>          Video driver ('Dummy').
```